wp-content/themes/csisnuclearnetwork/assets/_scss/components/_share.scss

```diff
@@ -46,6 +46,11 @@
     top: -2px;
   }
 
+  .a2a_button_x {
+    position: relative;
+    top: -2px;
+  }
+
```

Comment:
Since we're using the viewbox to position the icon, let's remove this and, if necessary, adjust the position in the viewbox.